### PR TITLE
Remove broken versions from index.yaml

### PR DESCRIPTION
**Pull request by aiyengar2**
This should have been done as part of https://github.com/rancher/charts/pull/975, but was a missed step.

Related Issue: https://github.com/rancher/rancher/issues/31137

### CODE REVIEW

**Review by deniseschannon — APPROVED** [review]

File: index.yaml [hunk 1]
```diff
@@ -1283,31 +1283,7 @@ entries:
       catalog.cattle.io/ui-component: rancher-backup
     apiVersion: v1
     appVersion: v1.0.3
-    created: "2021-01-16T02:08:30.685742727Z"
```
Comment (aiyengar2) [inline index.yaml:1286]: No longer exists

File: index.yaml [hunk 2]
```diff
@@ -1542,21 +1518,7 @@ entries:
       catalog.cattle.io/namespace: cis-operator-system
       catalog.cattle.io/release-name: rancher-cis-benchmark-crd
     apiVersion: v1
-    created: "2021-01-16T02:08:30.687265344Z"
```
Comment (aiyengar2) [inline index.yaml:1545]: no longer exists

**Comment by aiyengar2** [comment]
@deniseschannon I completely regenerated the index.yaml just to take care of any edge cases by using the following workflow:

```shell
arvindiyengar: ~/Rancher/charts/src/github.com/rancher/charts 
$ helm repo index --merge ./assets/index.yaml --url assets assets                           
WARNING: "kubernetes-charts.storage.googleapis.com" is deprecated for "stable" and will be deleted Nov. 13, 2020.
WARNING: You should switch to "https://charts.helm.sh/stable" via:
WARNING: helm repo add "stable" "https://charts.helm.sh/stable" --force-update
arvindiyengar: ~/Rancher/charts/src/github.com/rancher/charts 
$ cp ./assets/index.yaml ./released/assets/index.yaml 
arvindiyengar: ~/Rancher/charts/src/github.com/rancher/charts 
$ helm repo index --merge ./released/assets/index.yaml --url released/assets released/assets
WARNING: "kubernetes-charts.storage.googleapis.com" is deprecated for "stable" and will be deleted Nov. 13, 2020.
WARNING: You should switch to "https://charts.helm.sh/stable" via:
WARNING: helm repo add "stable" "https://charts.helm.sh/stable" --force-update
arvindiyengar: ~/Rancher/charts/src/github.com/rancher/charts 
$ cp ./released/assets/index.yaml index.yaml
```

File: index.yaml (hunk 2 above)
Comment (aiyengar2) [reply index.yaml:1545]: Root cause of https://github.com/rancher/rancher/issues/31137

**Pull request merged**